src/nrc/datamodel/admin.py

```diff
@@ -45,11 +49,112 @@ def get_object_actions(self, obj):
         )
 
 
+@admin.action(
+    description=_("Check the status of the callback URLs of selected Subscriptions")
+)
+def check_callback_url_status(modeladmin, request, queryset):
+    """
+    Make a request to the callback URLs of all selected Abonnementen and store the results
+    in the session
+
+    Any subsequent executions of this action will remove the previous results from the session
+    """
+    validator = CallbackURLValidator("callback_url", "auth")
+    callback_statuses = {}
+    for obj in queryset.iterator():
+        # Any other Abonnement that has the same callback URL and auth will be skipped
+        # and gets the same status
+        key = str((obj.callback_url, obj.auth))
```

Comment:
Why do you convert it to str? A tuple can also be a dict key, right?

Reply:
I tried this at first, it can be a dict key, but once you try to store it in the session it will complain about it not being serializable